rotkehlchen/globaldb/handler.py

```diff
@@ -1801,3 +1811,62 @@ def get_asset_type(identifier: str) -> AssetType:
             raise UnknownAsset(identifier)
 
         return AssetType.deserialize_from_db(type_in_db[0])
+
+    @staticmethod
+    def _resolve_asset_from_packaged_db(
+            identifier: str,
+            form_with_incomplete_data: bool,
+    ) -> AssetWithNameAndType:
+        """
+        This is a fallback method to resolve an asset.
+        It checks if the asset is in the packaged global db.
+
+        May raise:
+        - UnknownAsset
+        """
+        log.debug(f'Attempt to resolve asset {identifier} using the packaged database')
+        root_dir = Path(__file__).resolve().parent.parent
+        builtin_database = root_dir / 'data' / 'global.db'
+
+        query = """
+        SELECT A.identifier, A.type, B.address, B.decimals, A.name, C.symbol, C.started, null, C.swapped_for, C.coingecko, C.cryptocompare, B.protocol, B.chain, B.token_kind, null, null FROM builtin_db.assets as A JOIN builtin_db.evm_tokens as B
```

Comment:
Don't duplicate code, it should reuse the same logic as it is the same database schema